config/default/field.field.node.event.field_event_teaser.yml

```diff
@@ -0,0 +1,21 @@
+uuid: 295fff5d-7430-4806-821a-9663f9921402
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_event_teaser
+    - node.type.event
+  module:
+    - text
+id: node.event.field_event_teaser
+field_name: field_event_teaser
+entity_type: node
+bundle: event
+label: 'Event Teaser'
+description: ''
+required: false
+translatable: false
+default_value: {  }
+default_value_callback: ''
+settings: {  }
+field_type: text
```

Comment:
Please convert the event teaser to full-text multiline editor and move the field below the title.

![Screenshot from 2019-12-05 21-31-37](https://user-images.githubusercontent.com/47801537/70227415-a8aad780-17a6-11ea-8446-f30bc3c8df52.png)